### PR TITLE
[WIP] Prepping for merge upstream...

### PR DESCRIPTION
**Pull request by briehl**
Got a few things here:
1. Updated config.json to have a cdn link.
2. Started updating docs on the major changes.
3. Fixed issues with the updater on VERY old narratives.
4. Bumped the version to 3.0.0-alpha-1

Still missing:

- [x] Fix app cell code that prevented running
- [x] Finish updating release notes
- [ ] Get better stubs for documentation (docs will be ongoing the rest of this week, probably)
- [x] Map obsolete apps/methods -> SDK'd ones for those that have them.
- [x] Update config.json to point to CI CDN once that's up.

### CODE REVIEW

File: src/config.json [hunk 1]
```diff
@@ -88,6 +91,7 @@
         "awe": "http://140.221.67.185:7080",
         "catalog": "https://ci.kbase.us/services/catalog",
         "cdmi": "https://kbase.us/services/cdmi_api/",
+        "cdn": "https://ci.kbase.us/cdn",
```
Comment (kkellerlbl) [inline src/config.json]: What's the dev block used for?

Comment (briehl) [inline src/config.json]: That's pretty much a holdover from the pre-CI/Next/appdev days. We can probably lay it to rest now.

**Comment by briehl** [comment]
Oops, this adds a bug that prevents App cells from running. Don't merge yet!

**Comment by briehl** [comment]
Good as it's gonna get for now, I think...

**Pull request merged**